src/lib/timers/ingress_drop_monitor.lua

```diff
@@ -65,14 +83,21 @@ function IngressDropMonitor:jit_flush_if_needed ()
    if now() - self.last_flush < self.wait then return end
    self.last_flush = now()
    self.last_value[0] = self.current_value[0]
+   counter.set(self.ingress_packet_drop, self.last_value[0])
+
    --- TODO: Change last_flush, last_value and current_value fields to be counters.
    local msg = now()..": warning: Dropped more than "..self.threshold.." packets"
    if self.action == 'flush' then
       msg = msg.."; flushing JIT to try to recover"
    end
    msg = msg..". See "..self.tips_url.." for performance tuning tips."
    print(msg)
-   if self.action == 'flush' then jit.flush() end
+
+   self.ingress_packet_drop_alarm:raise({alarm_text=msg})
+   if self.action == 'flush' then
+      jit.flush()
+      self.ingress_packet_drop_alarm:clear()
```

Comment:
I think we should clear this alarm in the `if` statement that's on line 82 of this version.  WDYT?

Reply:
OK, makes sense.